### PR TITLE
add 977-Squares-of-Sorted-Array

### PR DESCRIPTION
**Pull request by Rwatana**
Please check my code

### CODE REVIEW

File: 977_squares_of_sorted_array.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from typing import List
+
+
+class Solution:
+    def sortedSquares(self, nums: List[int]) -> List[int]:
```
Comment (ryujumoto) [inline 977_squares_of_sorted_array.py:8]: This solution is intuitive, but you can solve this with more efficient solution.

**Review by ryujumoto — COMMENTED** [review]
Done.

**Comment by Rwatana** [comment]
I changed and very simple. Please check my code.

File: 977_squares_of_sorted_array.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# time: O(1)
+# space: O(1)
+from typing import List
+
+
+class Solution:
+    
+    def sortedSquares(self, nums: List[int]) -> List[int]:
+        return sorted([x**2 for x in nums])
```
Comment (ryujumoto) [inline 977_squares_of_sorted_array.py]: This solution is the same as your former solution, which was seeing each element and doubling it, then sorting the array. As I said before, it is inefficient and you can solve it more efficiently by using two pointers. Try the solution.

**Comment by Rwatana** [comment]
I revised the way that use two pointers. Please check my code.

**Review by ryujumoto — APPROVED** [review]

**Pull request merged**